src/ansys/templates/python/osl_solution/{{cookiecutter.__project_name_slug}}/src/ansys/solutions/{{cookiecutter.__solution_name_slug}}/solution/problem_setup_step.py

```diff
@@ -48,6 +50,8 @@ class ProblemSetupStep(StepModel):
     }
     step_list: list = []
     node_list: list = []
+    system_hierarchy: list = []
+    step_list: list = []
```

Comment:
@ansRMorin this field already exists at line 49.

Reply:
removed!